.pipelines/ci.yml

```diff
@@ -274,6 +274,103 @@ jobs:
           ctest -C RelWithDebInfo --output-on-failure
         displayName: Run C++ native tests
 
+  #############
+  # ARM64EC
+  #############
+
+  - job: ARM64ECBuild
+    pool:
+      vmImage: 'windows-2019'
```

Comment:
try this latest vm-image: https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

Reply:
sounds good, thanks!